docs/boot-options.rst

```diff
@@ -525,13 +555,20 @@ may connect.
 
 A system installed with VNC will start in text mode (runlevel 3).
 
+This option is deprecated and will be removed in future releases.
+Use ``inst.rdp`` instead.
+
+
 .. inst.vncpassword:
 
 inst.vncpassword
 ^^^^^^^^^^^^^^^^
 
 Set a password on the VNC server used by the installer.
 
+This option is deprecated and will be removed in future releases.
+Use ``inst.rdp`` and related boot options instead.
```

Comment:
I think here should be ``inst.rdp.password`` instead.

Reply:
Indeed, fixed. :)